socketio/examples/callback.rs

```diff
@@ -11,8 +11,11 @@ fn main() {
     // socket to communicate with the server
     let handle_test = |payload: Payload, socket: RawClient| {
         match payload {
-            Payload::String(str) => println!("Received string: {}", str),
+            Payload::Text(text) => println!("Received json: {:?}", text),
```

Comment:
nit: maybe switch formatting to `{:#?}` to be consistent with how we format bytes in the next line?